python/shotgun_fields/shotgun_field_editable.py

```diff
@@ -152,6 +159,13 @@ def eventFilter(self, obj, event):
 
         return False
 
+    def __getattr__(self, name):
```

Comment:
Since this one only wraps the display, maybe this is ok. I guess it should forward `value_changed` though.